### ISSUE TITLE
Teleposer crash?

### ISSUE THREAD

**Issue by TommyT88** [MrTJP/ProjectRed#862]
Hello there! A user of ours reported our server crashed when he put down a teleposer. 
Here's the crashlog

-----------------------------------------------------

[02:21:38] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoClassDefFoundError: mrtjp/core/fx/particles/BeamPulse2
	at mrtjp.projectred.expansion.TileTeleposer.<init>(TileTeleposer.scala:297) ~[TileTeleposer.class:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.7.0_72]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.7.0_72]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.7.0_72]
	at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:1.7.0_72]
	at java.lang.Class.newInstance(Unknown Source) ~[?:1.7.0_72]
	at mrtjp.core.block.InstancedBlock.createTileEntity(tileblock.scala:177) ~[InstancedBlock.class:?]
	at mrtjp.core.block.InstancedBlock.createTileEntity(tileblock.scala:130) ~[InstancedBlock.class:?]
	at net.minecraft.world.chunk.Chunk.func_150806_e(Chunk.java:850) ~[apx.class:?]
	at net.minecraft.world.chunk.Chunk.func_150807_a(Chunk.java:660) ~[apx.class:?]
	at net.minecraft.world.World.func_147465_d(World.java:451) ~[ahb.class:?]
	at net.minecraft.item.ItemBlock.placeBlockAt(ItemBlock.java:206) ~[abh.class:?]
	at mrtjp.core.block.ItemBlockCore.placeBlockAt(coreblock.scala:42) ~[ItemBlockCore.class:?]
	at net.minecraft.item.ItemBlock.func_77648_a(ItemBlock.java:103) ~[abh.class:?]
	at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505) ~[ForgeHooks.class:?]
	at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128) ~[add.class:?]
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389) ~[mx.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556) ~[nh.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60) ~[jo.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9) ~[jo.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) ~[ej.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165) ~[nc.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659) ~[MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334) ~[lt.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]
Caused by: java.lang.ClassNotFoundException: mrtjp.core.fx.particles.BeamPulse2
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]
	... 27 more
Caused by: java.lang.NoClassDefFoundError: mrtjp/core/fx/particles/CoreParticle
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.7.0_72]
	at java.lang.ClassLoader.defineClass(Unknown Source) ~[?:1.7.0_72]
	at java.security.SecureClassLoader.defineClass(Unknown Source) ~[?:1.7.0_72]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]
	... 27 more
Caused by: java.lang.ClassNotFoundException: mrtjp.core.fx.particles.CoreParticle
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.7.0_72]
	at java.lang.ClassLoader.defineClass(Unknown Source) ~[?:1.7.0_72]
	at java.security.SecureClassLoader.defineClass(Unknown Source) ~[?:1.7.0_72]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.7.0_72]

**Comment by TommyT88** [comment]
Additional info: Crash happened for the user when he put it down, he however appears to have gotten it back (after I fixed the server). Which explains why I couldn't find the machine in the region files. It appears to have left "ghost data" after crashing. The fist time I checked the area after rebooting I saw the machine, but when I right-clicked it crashed the server and every subsequent time I tried to log back in the server would crash again. The only fix I could do was rollback the region file from a backup.
